### PR TITLE
feat(starters): add Ueno to starters list (#10972)

### LINKED ISSUE #10972: Add Ueno's starter
It would be great to include Ueno's starter in a list of starters in the documentation.

### PR DESCRIPTION
**Pull request by birkir**
Add Ueno to starters list
Fixes #10972

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -1609,3 +1609,15 @@
     - Progressive Web app
     - Offline Support
     - RSS Feed
+- url: https://ueno.co/
+  repo: https://github.com/ueno-llc/ueno-gatsby-starter
+  description: Ueno's starter template for Gatsby
+  tags:
+    - Ueno
```
Comment (LekoArts) [inline docs/starters.yml]: Sorry, but we don't accept tags referencing single companies/agencies etc. They should stay meaningful and enable the user to have a good overview. So I kindly ask you to remove that 👍 

```suggestion
```

Comment (birkir) [reply docs/starters.yml]: Fixed 👍

File: docs/starters.yml [hunk 2]
```diff
@@ -1609,3 +1609,14 @@
...
     - RSS Feed
+- url: https://ueno.co/
+  repo: https://github.com/ueno-llc/ueno-gatsby-starter
+  description: Ueno's starter template for Gatsby
+  tags:
+    - TypeScript
+  features:
+    - GraphQL hybrid
+    - SEO friendly
+    - GSAP ready
```
Comment (DSchau) [inline docs/starters.yml]: Why is this GSAP ready?

File: docs/starters.yml [hunk 3]
```diff
@@ -1609,3 +1609,14 @@
...
+    - Nice devtools
```
Comment (DSchau) [inline docs/starters.yml]: I'm a little confused by this. Is this this devtools setup? What exactly is this doing?

https://github.com/ueno-llc/ueno-gatsby-starter/blob/master/src/components/devtools/GridOverlay.tsx

File: docs/starters.yml [hunk 4]
```diff
@@ -1609,3 +1609,14 @@
...
+    - Decorators
```
Comment (DSchau) [inline docs/starters.yml]: Are you using decorators?

File: docs/starters.yml (hunk 3 above)
Comment (birkir) [reply docs/starters.yml]: This is one of the devtools, we forgot to add our gasp tools, we will push an update tomorrow.

This particular devtools displays an overlay grid that uses configuration from scss styles folder. It has they same keyboard shortcuts as in Sketch.

File: docs/starters.yml (hunk 4 above)
Comment (birkir) [reply docs/starters.yml]: Yes, we have decorator support through our plugin: https://github.com/ueno-llc/gatsby-plugin-ueno/blob/4e9c14acce8dc3df0f76f29817dcaacfc7db38df/src/gatsby-node.js#L4

**Comment by DSchau** [comment]
@birkir fair point--but I also think this review uncovered some things that could be (and were!) improved--so thanks for being flexible.

If you can fix up the merge conflicts we can get this merged; thanks!

**Comment by birkir** [comment]
No problem, don't get me wrong. Always good to be thorough and I appreciate the feedback 👍

**Pull request merged**